### ISSUE TITLE
Move monitor to GCP

### ISSUE THREAD

**Issue by marcocastignoli** [ethereum/sourcify#1524]
* We need to move monitor to the new infra on GCP

**Comment by marcocastignoli** [comment]
I moved monitor to GCP but there seems to be several problems:
* communication issues between between monitor and server (it happens apparently randomly): 
  ```
  {"address":"0x4bD4425E2E3AEf4F579bd74FA78831525A0BD110","error":{"cause":{"code":"ECONNRESET","host":"sourcify-staging-server-lidblonrca-ew.a.run.app","path":null,"port":443},"code":"ECONNRESET","config":{"adapter":["xhr","http","fetch"],"data":"..."}
  ```
  * I thought it could be the "min number of instances = 0" settings for Cloud Run Services for which Sourcify server took too long to reply because it had to start a new instance before receiving a request from monitor. But it doesn't seem to be the case
  * I'm using `axios` in monitor instead of `fetch` because otherwise errors are not logged
* block polling fails too many times leaving monitor behind.

Todo:
* Use filebase private gateway instead of a public IPFS gateway

**Comment by marcocastignoli** [comment]
I published both monitor-staging and monitor-production on GCP.

I didn't find a solution for the `ECONNRESET` error explained above. But I implemented a retry mechanism #1561 that should mitigate the problem.
